chore: Losslessly optimize raster images

anthonyryan1 opened the pull request:
### What

Using [Efficient-Compression-Tool](https://github.com/fhanau/Efficient-Compression-Tool), we can reduce the size of site images by 15.01 MB (20.65%) without changing the visual appearance.

The cumulative effect of these image savings will slightly reduce web server bandwidth and make the site a little more accessible for people on slow connections.

hangy reviewed (approved):
Thanks! I think some images that you updated are actually unused, but that should be resolved elsewhere and is out of scope. Good idea!

anthonyryan1 committed:
chore: Losslessly optimize raster images

Using Efficient-Compression-Tool, we can reduce the size of site
images by 15.73 MB (20.4%) without changing the visual appearance.
